docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -155,6 +155,17 @@ it is terminated. The time includes time for analysis and planning, but also
 time spend in a queue waiting, so essentially this is the time allowed for a
 query to exist since creation.
 
+``query.max-scan-physical-bytes``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-data-size`
+* **Default value:** ``empty``
```

Comment:
I think we just leave this out .. basically there is no default value .. so there is no defined max value ..

Reply:
``empty`` is not a valid value by itself